### DISCUSSION TITLE
Needs some more tutorial on how to use PaLM, Anthropic, and other cloused models in spacy-llm

### DISCUSSION THREAD

**Discussion by innocent-charles** [explosion/spacy-llm#345] [General]
Needs some more tutorial on how to use PaLM, Anthropic, and other cloused models in spacy-llm

**Comment by rmitsch** [comment]
Hi @innocent-charles, what exactly are you struggling with? See e. g. the [usage docs for examples](https://spacy.io/usage/large-language-models#example-1) on how to use those models.

**Comment by innocent-charles** [comment]
There is usage on models from Open AI. The idea is i have the .env that i put access and load it using load_dotenv() from from dotenv import load_dotenv, this works fine when the OPENAI_API_KEY="" in the .env , so was asking if i want to use PaLM forexample, should i put api_key="  " ?. i have tried that but told   File "/home/innocent/.local/lib/python3.11/site-packages/spacy_llm/models/rest/palm/model.py", line 29, in credentials
    assert api_key is not None
           ^^^^^^^^^^^^^^^^^^^

**Reply by rmitsch** [reply]
Have a look at the section on [API keys in the docs](https://spacy.io/api/large-language-models#api-keys). Does this answer your question?

**Comment by innocent-charles** [comment]
Yes.. It answer my question perfectly. Thanks much @rmitsch

**Comment by innocent-charles** [comment]
Thanks it now works fine.
